app/services/job_posting_api/current_job_postings_poller.rb

```diff
@@ -56,7 +56,10 @@ def process_feed(feed)
     end
 
     def sync_posting(job_posting, attributes)
-      job_posting.assign_attributes(attributes.merge(DEFAULT_ATTRIBUTES))
+      enhanced_attributes = attributes.merge(**DEFAULT_ATTRIBUTES)
+      enhanced_attributes[:address_attributes][:id] = job_posting.address_id if job_posting.address.present?
```

Comment:
Bit ugly but I haven't had another idea how to fix the issue yet.
Since it's only a nested attribute, to update we need the id of the address. Other solution would be to just destroy the current address and insert a new one, but this would cause two queries to be issued which is not tooo optimal for performance (even though performance is not such a bit issue yet since it's in a background worker anyways and we do not have thousands of job postings anyway yet)